src/pages/CreateAssignment/CreateAssignment.js

```diff
@@ -87,34 +106,45 @@ function CreateAssignment({ firebase }) {
       />
       {(showWriting || showVocab || showPhonics) && (
         <div>
-          <h1>Create Assignment</h1>
-          {(showWriting || showVocab) && (
-            <WordGroupSelector
-              handleChange={handleWordSelectorChange}
-              wordGroupChange={handleWordGroupChange}
-            />
-          )}
-          <br />
-          <div className="spacing"></div>
-          <div className="place_middle">
-            <Container>
-              <Row>
-                <Col>
-                  <StudentList
-                    deployments={adminDeployments}
-                    handleChange={handleStudentListChange}
-                  />
-                </Col>
-                <Col xs={1}></Col>
-                <Col>
-                  <DatePicker handleChange={handleDatePickerChange} />
-                </Col>
-              </Row>
-            </Container>
+          <div className="assignment_creation">
+            <h1>Create Assignment</h1>
+            {(showWriting || showVocab) && (
+              <WordGroupSelector
+                handleChange={handleWordSelectorChange}
+                wordGroupChange={handleWordGroupChange}
+              />
+            )}
+            <br />
+            <div className="spacing"></div>
+            <div className="place_middle">
+              <Container>
+                <Row>
+                  <Col>
+                    <StudentList
+                      deployments={adminDeployments}
+                      handleChange={handleStudentListChange}
+                    />
+                  </Col>
+                  <Col xs={1}></Col>
+                  <Col>
+                    <DatePicker handleChange={handleDatePickerChange} />
+                  </Col>
+                </Row>
+              </Container>
+            </div>
+            <Button onClick={() => validateAssignment()} className="assign">
```

Comment:
for `onClick` functions that you don't have to pass in parameters for, you can just do `onClick = {functionName}` without the parentheses- this `() => functionName(param)` hack is only for when you're passing in params